src/reducers/tokens/index.js

```diff
@@ -0,0 +1,73 @@
+import { handleActions } from 'redux-actions'
+import reduceReducers from 'reduce-reducers'
+
+import { tokens } from '../../actions/tokens'
+
+const initialState = {
+    likelyContracts: [],
```

Comment:
Right, but I'm not sure about it, because when I start refactoring it, it was like that:
https://github.com/near/near-wallet/blob/master/src/components/wallet/Wallet.js#L169

So `likelyTokens` endpoint becomes `likelyContracts`, I thought that's weird (like you), and started using `likelyTokens` everywhere, you can see it at my first commits - but after that, it was much more confusing because next we are using `contract` (instead of `token`) everywhere - so I changed it back to `likelyContracts`.

So I'm not sure how to handle it, we can use `likelyTokens`, but then we should use `tokens` instead of `contract` but that also confusing since it's not a token :) Maybe we should change the endpoint name, at some point?

Reply:
It sounds like the sanest thing to do is re-name `likelyTokens` everywhere, including the `near-contract-helper` route, to be `/likelyTokenContracts`? :).

If that sounds correct, could you create an issue for that change? I'm happy to add a  route for`/likelyTokenContracts` to `near-contract-helper` that is bound to the same route handler as `/likelyTokens`, then once that is landed we can rename the call to the route as a separate PR to `near-wallet`.

For this PR, let's rename everything to use the naming convention we want the contract-helper route to be (`likelyTokenContracts`?) :). That way the follow-up PR after we've added the new route to the contract-helper will only be a change to that one line that performs the API call